src/main/java/StudentManagementSystem.java

```diff
@@ -0,0 +1,55 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class StudentManagementSystem {
+    private UserManagement userManagement;
+    private RegistrationService registrationService;
```

Comment:
Inside of the `RegistractionService`, you are using the `LoginService` object. Then inside this class, you have both classes included. This is a lot of coupling and we should aim to separate our classes more.